lxqt-base/lxqt-session/lxqt-session-0.12.0.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+inherit cmake-utils
+
+DESCRIPTION="LXQT session manager"
+HOMEPAGE="http://lxqt.org/"
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/lxde/${PN}.git"
+else
+	SRC_URI="https://github.com/lxde/${PN}/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+fi
+
+LICENSE="GPL-2 LGPL-2.1+"
+SLOT="0"
+
+RDEPEND="
+	>=dev-libs/libqtxdg-1.0.0:=
+	dev-qt/qtcore:5=
+	dev-qt/qtdbus:5=
+	dev-qt/qtgui:5=
+	dev-qt/qtwidgets:5=
+	dev-qt/qtx11extras:5=
+	dev-qt/qtxml:5=
+	kde-frameworks/kwindowsystem:5=[X]
+	~lxqt-base/liblxqt-${PV}
+	x11-libs/libX11
+	x11-misc/xdg-user-dirs
+	!!lxqt-base/lxqt-common
+"
+DEPEND="${RDEPEND}
+	dev-qt/linguist-tools:5=
+	>=dev-util/lxqt-build-tools-0.4.0
+	dev-util/intltool
+	sys-devel/gettext
+	virtual/pkgconfig"
+
+src_configure() {
+	local mycmakeargs=( -DPULL_TRANSLATIONS=OFF )
+	cmake-utils_src_configure
+}
+
+src_install(){
+	cmake-utils_src_install
+	doman lxqt-config-session/man/*.1 lxqt-session/man/*.1
```

Comment:
``` c++

	dodir "/etc/X11/Sessions"
	dosym  "../../../../usr/bin/startlxqt" "/etc/X11/Sessions/lxqt"
```

Reply:
Could you please describe why this change is needed?

Reply:
These lines adds lxqt to the session list, I am using sddm and this was requied. I am quite sure it is also needed for other *dm. This was done in lxqt-common before, see https://github.com/gentoo/gentoo/blob/aff5eaa946f533f5e199d2d32e3279a505102280/lxqt-base/lxqt-common/lxqt-common-0.11.0.ebuild#L33

Reply:
I'm also using sddm (version 0.15.0) and even without those lines I'm able to switch between lxqt and kde5 sessions.

I think it's probably due to files installed in `/usr/share/xsessions` directory.

Reply:
You are right, it was useless. I tried removing them and sddm list lxqt just fine. My bad.